### PR TITLE
Remove non fractal type imports

### PR DESCRIPTION
**Pull request by adamgall**
For all imports from `@fractal-framework/fractal-contracts` that are not part of the actual Fractal contracts codebase (IVotes, ERC20, ERC721), remove those imports and either use newly created ABIs or the ABIs available from viem.

Also, clean up / remove some random unrelated code that I ran across.

### CODE REVIEW

File: src/hooks/DAO/loaders/governance/useERC20LinearToken.ts
Comment (adamgall) [inline src/hooks/DAO/loaders/governance/useERC20LinearToken.ts:1]: `isDelegatesSet` property in state is never used anywhere, so we're removing it.

File: src/hooks/DAO/loaders/governance/useLockRelease.ts
Comment (adamgall) [inline src/hooks/DAO/loaders/governance/useLockRelease.ts:1]: `isDelegatesSet` property in state is never used anywhere, so we're removing it.

File: src/providers/App/governance/reducer.ts
Comment (adamgall) [inline src/providers/App/governance/reducer.ts:1]: `isDelegatesSet` property in state is never used anywhere, so we're removing it.

File: src/types/account.ts
Comment (adamgall) [inline src/types/account.ts:1]: `isDelegatesSet` property in state is never used anywhere, so we're removing it.

File: src/types/fractal.ts
Comment (adamgall) [inline src/types/fractal.ts:1]: `isDelegatesSet` property in state is never used anywhere, so we're removing it.

File: src/assets/abi/IVotes.ts
Comment (adamgall) [inline src/assets/abi/IVotes.ts:1]: Pull the IVotes ABI into the codebase as const.

File: src/components/DaoCreator/hooks/usePrepareFormData.ts
Comment (adamgall) [inline src/components/DaoCreator/hooks/usePrepareFormData.ts:1]: Simple change here, just use viem style contract for IVotes function calls.

File: src/hooks/DAO/proposal/useUserERC721VotingTokens.ts
Comment (adamgall) [inline src/hooks/DAO/proposal/useUserERC721VotingTokens.ts:1]: This code in here was just leaving me scratching my head. Certainly wasn't doing what it said it was doing. Also, didn't cause any errors when I removed it so /shrug

File: src/types/contract.ts
Comment (adamgall) [inline src/types/contract.ts:1]: DAOContracts isn't used anywhere, removing it.

**Review by DarksightKellar — APPROVED** [review]
🚀

**Pull request merged**